atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,241 @@
+        for index in range(len(medoids)):
+            cluster = { "properties": [{
+                "cluster_id": str(uuid.uuid4())
+                }]}
+            self.weaviate_client.add_embeddings(CollectionNames.CLUSTERCENTER.value, medoids[index].tolist(), cluster)
+
+        competencies = self.weaviate_client.get_all_embeddings(CollectionNames.COMPETENCY.value)
+        clusters = self.weaviate_client.get_all_embeddings(CollectionNames.CLUSTERCENTER.value)
+
+        for competency in competencies:
+            competency_id, embedding = generate_embeddings(competency["properties"]["competency_id"], competency["properties"]["text"])
+            similarity_score = np.array(compute_cosine_similarity(embedding, medoid) for medoid in medoids)
```

Ruff (0.11.9)

Line too long (107 > 88)

(E501)